### ISSUE TITLE
Intercity passenger train lines

### ISSUE THREAD

**Issue by quincylvania** [ZeLonewolf/openstreetmap-americana#745]
Currently all railroads are drawn the same, but active passenger trains lines are of greater interest to a general audience. In North America there are relatively few passenger rail lines, especially compared to roads, so highlighting them seems doable. Some paper maps show passenger rail lines but no freight lines at all (see #101). For now I think we can focus on intercity and possibly commuter/regional routes. Metro/light rail/streetcar/heritage lines should be kept in mind in terms of styling hierarchy but might be future work.

Some ideas:
- Bolder styling for passenger rail lines (part of `route=train` relations)
- Show these lines at lower zooms
- Label the lines with the route names and/or operators
- Render associated train stations at higher zooms (see also #435)

**Comment by 1ec5** [comment]
Subway and light rail lines already receive special treatment at higher zoom levels, so distinguishing “consumer-friendly” heavy rail would be entirely consistent with this style, even if it isn’t heavily focused on public transportation per se.

> Label the lines with the route names and/or operators

I think the most typical treatment is to label the line with the route name, if at all, and also append the service’s logo to the stations’ labels. Rail operators as OSM knows them don’t tend to be labeled at all on consumer-oriented maps.

**Comment by 1ec5** [comment]
Labeling cities based on their long-distance rail and bus services would be an interesting data modeling problem. Relating train stations to cities could be a maintenance nightmare, similar to the ill-begotten [`destinationsign` relation type](https://wiki.openstreetmap.org/wiki/Relation:destinationsign). Wikidata has a [place served by transport hub <small>P931</small>](https://www.wikidata.org/wiki/Property:P931) property that’s well-suited for this purpose, but OpenMapTiles would need to join places and POIs on Wikidata; I don’t know if there’s any precedent for that yet.

**Comment by quincylvania** [comment]
> Subway and light rail lines already receive special treatment at higher zoom levels

Oh okay I see it now. I think this could be improved but that's a separate issue.

> Append the service’s logo to the cities’ labels at low zoom levels

Sounds cool but I wouldn't do this for Americana. Maybe one day we could have a transit-oriented variant.

> Rail operators as OSM knows them don’t tend to be labeled at all on consumer-oriented maps.

I guess I meant more the network. Like some rail lines host a dozen Amtrak routes and we wouldn't want to list all their names, but we can label the line "Amtrak". Apple Maps does something like this. I'm not suggesting we label "Keolis" or whatever.

**Comment by 1ec5** [comment]
>> Append the service’s logo to the cities’ labels at low zoom levels
> 
> Sounds cool but I wouldn't do this for Americana. Maybe one day we could have a transit-oriented variant.

I agree that a transit-oriented style would emphasize these details more heavily, but public transportation indicators do appear frequently in road maps too. For example, INDOT’s _Indiana Roadway Map 2018_ labels cities and towns by whether they have a “transit system” (red for large urban, black for small or rural). These icons indicate availability, not the precise locations of bus stations. This map shows mainline railways but only labls them by their freight operators, not services like Amtrak.

<img src="https://user-images.githubusercontent.com/1231218/215561205-dc84d62e-81b8-4a1f-a939-8bff6b74fd30.jpg" width="600" alt="INDOT 2018">

The general approach of labeling cities by their amenities isn’t unheard of either. Here’s a map of Missouri that indicates which cities have hospitals:

<img width="287" alt="MoDOT 2009" src="https://user-images.githubusercontent.com/1231218/215556647-9d7d4eb5-f8ba-4215-aa63-88db296de4d3.png">

Benchmark Maps even includes nifty climate charts in its maps of Western states, where that kind of information comes in handy:

<img width="279" alt="Benchmark" src="https://user-images.githubusercontent.com/1231218/215561694-18c32590-c185-48b8-831b-6a8415c62b28.png">

That said, I recognize there’s less need for this approach in an interactive map in which you could zoom in on a place or click on it to get more information.
